webpack/assets/javascripts/react_app/components/AuditsList/ExpansiveView.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { renderTemplatesDiff } from '../../../foreman_editor';
+import DiffContainer from '../DiffView/DiffContainer';
 import { translate as __ } from '../../common/I18n';
```

Comment:
we can remove this line as  I think we don't require it in this file.

Reply:
@ohadlevy,
I can see that translation used for actions like Diff, Preview on template page.
Do we need to apply translate about 'split', 'unified'?
I think yes.

Reply:
@kgaikwad we use it to translate "Comments" line 78.
good point on the split, unified. ill translate them